internal/generator/vector/conf_test.go

```diff
@@ -27,7 +27,7 @@ var _ = Describe("Testing Complete Config Generation", func() {
 		f = func(testcase testhelpers.ConfGenerateTest) {
 			g := generator.MakeGenerator()
 			if testcase.Options == nil {
-				testcase.Options = generator.Options{generator.TlsProfileSpec: tls.GetTLSProfileSpec(nil)}
+				testcase.Options = generator.Options{generator.ClusterTLSProfileSpec: tls.GetTLSProfileSpec(nil)}
```

Comment:
Why rename this? The string is still "tlsProfileSpec".

Reply:
To make it more obvious which profile it represents.  the actual string is immaterial